app/forms/hydranorth/forms/generic_file_edit_form.rb

```diff
@@ -5,12 +5,12 @@ class GenericFileEditForm < GenericFilePresenter
       include HydraEditor::Form
       include HydraEditor::Form::Permissions
       self.required_fields = [:title, :resource_type, :language, :creator, :subject, :license, :belongsToCommunity ]
-      
+
 
       # This is required so that fields_for will draw a nested form.
       # See ActionView::Helpers#nested_attributes_association?
       #   https://github.com/rails/rails/blob/a04c0619617118433db6e01b67d5d082eaaa0189/actionview/lib/action_view/helpers/form_helper.rb#L1890
-      def permissions_attributes= attributes
+      def permissions_attributes=(attributes)
```

Comment:
They're not ;). I was debugging in that file and cleaned up the style while I was at it.

Generally speaking I think it's better if we're consistent about whether we use them or not, and most of the code base seems to use them.